src/test/resources/simple/not-allowed-url-bag/fetch.txt

```diff
@@ -0,0 +1 @@
+ftp://dans.knaw.nl/nl/over/organisatie-beleid/informatiemateriaal/Software_Sustainability_DANS_NLeSC_2016.pdf  555292  data/brochure/Software_Sustainability_DANS_NLeSC_2016.pdf
```

Comment:
**General remark about the test resources:** given the large number of examples/bags/revisions/etc. would it be smart to make some kind of `README.md` in `src/test/resources/` with a small description of each example/bag/revision, what it is used for and which tests make use of each instance?